TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -335,7 +335,7 @@ def makeTreeFromMiniAOD(self,process):
                 subjetBTagDiscriminators = ['pfCombinedInclusiveSecondaryVertexV2BJetTags'],
                 JETCorrLevels = levels,
                 subJETCorrLevels = levels,
-                addEnergyCorrFunc = False,
+                addEnergyCorrFunc = True,
```

Comment:
Not sure if it's okay with you guys to have ECF's on by default, otherwise I'll make an option for this.